src/connector.rs

```diff
@@ -616,6 +626,37 @@ impl EthConnectorContract {
     pub fn set_paused_flags(&mut self, args: PauseEthConnectorCallArgs) {
         self.set_paused(args.paused_mask);
     }
+
+    /// Return metdata
+    pub fn get_metadata(&self) {
+        let icon = if let Some(ref icon) = self.metadata.icon {
+            format!(r#""{}""#, icon)
+        } else {
+            "null".to_string()
+        };
+        let reference = if let Some(ref reference) = self.metadata.reference {
+            format!(r#""{}""#, reference)
+        } else {
+            "null".to_string()
+        };
+        let reference_hash = if let Some(ref reference_hash) = self.metadata.reference_hash {
+            format!("{:?}", reference_hash)
```

Comment:
The reference implementation [encodes as base64 string](https://github.com/near/near-sdk-rs/blob/master/near-contract-standards/src/fungible_token/metadata.rs#L15). We should do the same.

Reply:
I'm not sure what do you mean. We receive as input parameter Base64 string, and return for ft_metadata - Base64 .
string. We don't need to encode/decode it.